tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -1584,7 +1584,7 @@
           <xsl:value-of select="$typename" />
           <xsl:text>",</xsl:text>
         </xsl:if>
-        <xsl:text>"type":"object","properties":{"results":{</xsl:text>
+        <xsl:text>"type":"object","properties":{"__count":{"type":"string","description":"The count name/value pair represents the number of entities in the collection. Available when using the [$inlinecount](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-odata/2ec0ff66-fd25-42cf-a37f-1317b085fecd) query option."},"results":{</xsl:text>

```

Comment:
I couldn't find the odata v2 specs on the http://docs.oasis-open.org site so found this one instead

https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-odata/2ec0ff66-fd25-42cf-a37f-1317b085fecd

Reply:
Here's the only publicly available version known to me:

https://help.sap.com/doc/5890d27be418427993fafa6722cdc03b/Cloud/en-US/OdataV2.pdf#page=67

Which is also used for the other system query options.